src/Components/TodoContext/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,32 @@
+import { useContext } from 'react';
+import { TodoContext } from '../TodoContext';
+import { TodoListItem } from '../../Todo/TodoListItem';
+import { Todo } from '../../../types/Todo';
+
+type FilterSettings = string;
+
+function filterList(list: Todo[], settings: FilterSettings): Todo[] {
+  return [...list].filter(item => {
+    switch (settings) {
+      case 'active':
+        return item.completed === false;
+      case 'completed':
+        return item.completed === true;
+      default:
+        return list;
+    }
+  });
+}
+
+export const ListOfTodo = () => {
```

Comment:
Consider renaming to TodoList to make one component name style the whole project